code/modules/unit_tests/monkey_business.dm

```diff
@@ -0,0 +1,42 @@
+/**
+ * Monkey Business
+ *
+ * This unit test spawns a predefined number of monkies, each of which
+ * are set to have a 100% chance of attempting to use something next to them each Life
+ *
+ * This test basically just checks to see if attack procs are working correctly,
+ * but its also hilarious and fun to watch locally.
+ */
+/datum/unit_test/monkey_business
+	priority = TEST_DEFAULT + 1 // should be the last test to run, except for create and destroy
+	var/monkey_count = 50
+	var/monkey_timer = 30 SECONDS
+	var/start_runtimes = 0
+	var/list/monkey_list
+	var/running = TRUE
+
+/datum/unit_test/monkey_business/Run()
+	monkey_list = list()
+	start_runtimes = GLOB.total_runtimes
+	for(var/monkey_id in 1 to monkey_count)
+		var/turf/spawn_turf = get_safe_random_station_turf()
```

Comment:
This needs to be consistent, otherwise it becomes even harder to reproduce failures.

Spawn 1 in every room at a consistently defined position, such as the first open floor tile.